recipe/conda_build_config.yaml

```diff
@@ -7,8 +7,11 @@ tempest:
   - notempest
   - tempest
 
+metis:
+  - 5.1.0
+
```

Comment:
We will want to undo this in the next build.  I would just do that immediately once this build is done but I'm not keen on building 50 more redundant packages.